dev/com.ibm.ws.jsp/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -23,7 +23,7 @@ extendedDocumentRoot.name=Extended document root
 extendedDocumentRoot=Directory that the JSP engine will search for additional JSP files to serve.
 
 javaSourceLevel.name=Java source level
-javaSourceLevel=Specifies the default Java source level for JSPs that are compiled by the JSP engine. Accepts one of the following LTS releases as a value: 8, 11, 17, and 21.
+javaSourceLevel=Specifies the default Java source level for JSPs that are compiled by the JSP engine. Accepts one of the following LTS releases as a value: 8, 11, 17, and 21. Note: jdkSourceLevel is supported, but it's use is discouraged.
```

Comment:
Not sure how to word this. Perhaps say "support still exists"?

Reply:
I don't like referencing an `internal` metatype from a public one. We stopped documenting the `jdkSourceLevel` to promote the use of `javaSourceLevel`